v/vert.x/vertx_ubi_8.5.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash -e
+# -----------------------------------------------------------------------------
+#
+# Package       : vert.x
+# Version       : 4.3.7
+# Source repo   : https://github.com/eclipse-vertx/vert.x
+# Tested on     : UBI 8.6
+# Language      : JAVA
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Shreya Kajbaje <Shreya.Kajbaje@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_NAME=vert.x
+PACKAGE_VERSION=${1:-4.3.7}
+PACKAGE_URL=https://github.com/eclipse-vertx/vert.x.git
+
+yum update -y
+yum install git wget  gcc gcc-c++ openssl  -y
+dnf install java-1.8.0-openjdk-devel -y
+
+dnf -y install maven
+
+if ! git clone $PACKAGE_URL; then
+    echo "------------------$PACKAGE_NAME:clone_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    exit 0
+fi
+
+cd $PACKAGE_NAME
+
+git checkout $PACKAGE_VERSION
+
+if ! mvn package; then
+    echo "------------------$PACKAGE_NAME:build_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    exit 1
+fi
+
+if ! mvn test; then
+    echo "------------------$PACKAGE_NAME:build_fails---------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    exit 2
+fi
+
+mvn -Dtest="Http1xProxyTest" surefire:test
```

Comment:
As I mentioned in one-one chat. Can we not run below tests in loop and also trap failure with return code as 2. Also if test results are very flaky, do we need to include those in build script? As random behavior of this will make currency maintenance tough
```mvn -Dtest="Http1xMetricsTest" surefire:test
mvn -Dtest="Http2TracerTest" surefire:test
mvn -Dtest="MetricsTest" surefire:test
mvn -Dtest="MetricsContextTest" surefire:test
mvn -Dtest="StartStopListCommandsTest" surefire:test
mvn -Dtest="RunCommandTest" surefire:test
```
`mvn test -PtestNativeTransport`  or `mvn test -PtestDomainSockets` will not run in `mvn test`?